### DISCUSSION TITLE
[Notice] Community Weekly Meeting (Mar 2, 2022)

### DISCUSSION THREAD

**Discussion by tzssangglass** [apache/apisix#6470] [General]
i, here is the schedule for the coming Weekly Meeting.

Schedule

Date: on Wednesday, Mar 2, at 20:00 ~ 22:00 (UTC +8)

Meeting Link: https://meeting.tencent.com/dm/F8kPP9xmDvBP

Content:

Apache APISIX is one of the active API Gateway projects in the world. Since
open source on June 6, 2019, it has accumulated more than 300 contributors.
The first open source contributions of many contributors were made on
Apache APISIX. We invited these lovely contributors to chat with you about
how to get involved in the open source community.

In this meeting we will discuss APISIX's V3 (2022) Roadmap.

Mailing list for APISIX's V3 (2022) Roadmap: https://lists.apache.org/thread/n6xd1d7v5jv3p9brb6fo878yl0nmrdnv

The doc of Chinese version: https://docs.qq.com/doc/DQ2ZxWHdQb3NtRVhQ

Note:

If you have any questions, please comment below or contact membphis@apache.org or
tzssangglass@apache.org .
Kindly let me know if you need Google Meet or Zoom as communication tools.

**Comment by juzhiyuan** [comment]
@membphis any plan to have next meeting?

**Reply by membphis** [reply]
Yes, we'll discuss `How to join the APISIX community` next week.
@tzssangglass will send the Notice later
